app/assets/graphing/graph.min.js

```diff
@@ -1 +1 @@
able>")+"</div>":""}},config.point={r:function(t){return data.radii[t.id]?30*data.radii[t.id][t.index]/data.maxRadii[t.id]:0}},config.axis.x.tick&&!config.axis.x.tick.format&&(config.axis.x.tick.format=function(t){var a=String(t);if("time"===type){var e=a.match(/\d+:\d+:\d+/)[0];a=t.getFullYear()+"-"+(t.getMonth()+1)+"-"+t.getDate()+" "+e}var i=void 0===axis.xLabels[a]?t:axis.xLabels[a];return Math.round(i)||i}),config.axis.y.tick&&(config.axis.y.tick.format=function(t){return axis.yLabels[String(t)]||Math.round(t)}),config.axis.y2.tick&&(config.axis.y2.tick.format=function(t){return axis.y2Labels[String(t)]||Math.round(t)});var e=[];for(var i in config.data.xs)data.isData[i]||e.push(i);config.legend.hide=e;var r=!!config.data.labels;config.data.labels={format:function(t,a,e){return r&&data.isData[a]?t||"":data.annotations[a]||""}},config.data.order=!1,config.onrendered=function(){for(var t in data.pointStyles){var a=data.pointStyles[t];applyPointShape(t,a),applyLegendShape(t,a)}for(var t in config.data.colors){if("bar"===type)for(var e=d3.selectAll(".c3-bars-"+t+" path")[0],i=0;i<e.length;i++){var r;data.barColors[t]&&data.barColors[t][i]&&(e[i].style.fill=data.barColors[t][i]),data.barOpacities[t]&&(r=data.barOpacities[t][i]),r=r||data.lineOpacities[t],e[i].style.opacity=r}else{var l=d3.selectAll(".c3-lines-"+t+" path")[0][0];l&&(l.style.opacity=data.lineOpacities[t])}var n=d3.selectAll(".c3-legend-item-"+t+" path")[0];n.length||(n=d3.selectAll(".c3-legend-item-"+t+" line")[0]),n.length&&((n=n[0]).style.opacity=data.lineOpacities[t],n.style.stroke=data.barColors[t][0]);var o=d3.selectAll(".c3-circles-"+t+" path")[0];o.length||(o=d3.selectAll(".c3-circles-"+t+" circle")[0]);for(i=0;i<o.length;i++)o[i].style.opacity=data.lineOpacities[t]}for(var t in data.areaColors){var c=d3.selectAll(".c3-areas-"+t+" path")[0][0];c&&(c.style.fill=data.areaColors[t],c.style.opacity=data.areaOpacities[t])}var d=0,s=setInterval(function(){d+=retryFrequency,window.Android?(clearInterval(s),Android.run()):d>maxDelay&&clearInterval(s)})},c3.generate(config)}catch(t){displayError(t)}}else delay>maxDelay&&clearInterval(intervalID)},retryFrequency);function applyPointShape(t,a){if("bar"!==type&&"bubble"!==type){var e=d3.selectAll(".c3-circles-"+t),i=e.selectAll("circle")[0];if(i)for(var r=0;r<i.length;r++)i[r].style.opacity=0,appendSymbol(e,i[r].cx.baseVal.value,i[r].cy.baseVal.value,a,i[r].style.fill)}}function applyLegendShape(t,a){if("none"!==a){var e=d3.selectAll(".c3-legend-item-"+t),i=e.selectAll("line");if(!i||!i.length)return;(i=i[0][0]).style.opacity=0,appendSymbol(e,i.x1.baseVal.value+5,i.y1.baseVal.value,a,i.style.stroke)}}function appendSymbol(t,a,e,i,r){"none"!==i&&t.append("path").attr("transform",function(t){return"translate("+a+", "+e+")"}).attr("class","symbol").attr("d",d3.svg.symbol().type(i).size(50)).style("fill",r).style("stroke",r)}function displayError(t){console.log(t);var a=document.getElementById("error");a.innerHTML=t,a.style.display="block"}
```

Comment:
I'm surprised this file has changed so much. Verifying that you tested it? Also curious what you used to generate this file. It might be good to always use the same tool so we don't get non-meaningful changes due to differences in minifiers.

Reply:
I used a random online minifier. I tested this though agree that good to use the same tool to minimize this diff. Is there a standard tool that we use across Dimagi, would prefer to re-minify this using that.

Reply:
I don't think there's a standard - on HQ we use Django's compressor, which isn't practical for the mobile team to use here. I've also just used random online minifiers - I think usually https://www.cleancss.com/css-minify/ but not for any particular reason.

Could you you pick your favorite online minifier and document it, probably [here](https://github.com/dimagi/commcare-core/blob/master/src/main/java/org/commcare/core/graph/model/GraphData.java#L101-L103)? And maybe also where that code is duplicated in [formplayer](https://github.com/dimagi/formplayer/blob/master/src/main/java/util/FormplayerGraphUtil.java#L44-L46). Then we'll have a standard.